<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <!--
    human_control.launch
    VERSION 0.0.0 Last Changed 2019-10-12
    Launch joystick to canbus control. This file should not be run directly- instead, run one of the main_* scripts
    which call this one
  -->
  <!-- Interface: 
    Pub: 
      sent_messages (can_msgs/Frame)

    Sub:
  -->

  <node pkg="rovr_control" type="joy_to_onboard_direct_node" name="joy_to_onboard_direct_node" output="screen">
    <param name="frequency" value="2.0"/>
    <param name="linear_scale" value="1.0"/>
    <param name="angular_scale" value="1.0"/><!--
    <param name="lift_scale_up" value="0.5"/>
    <param name="lift_scale_down" value="0.45"/>-->
    <param name="trans_conv_scale" value="1.0"/>
    <param name="digger_scale" value="1.0"/>
    <param name="hold_conv_scale_fwd" value="1.0"/>
    <param name="hold_conv_scale_back" value="0.5"/>
  </node>

  <node pkg="socketcan_bridge" type="socketcan_bridge_node" name="socketcan_bridge_node" output="screen"/>
</launch>
